src/components/Carousel/Carousel.js

```diff
@@ -111,18 +132,41 @@ class Carousel extends Component {
     });
   }
 
+  renderChildren() {
+    const { children, items } = this.props;
+    const { widthOfSlide } = this.state;
+    const style = { maxWidth: `${widthOfSlide}px` };
+    const listOfChildren = [...children];
+
+    return listOfChildren.map((item, i) => (
+      <div
+        key={items[i].id}
+        className={`${CN}__child`}
+        style={style}
+      >
+        {item}
+      </div>
+    ));
+  }
+
+
   render() {
-    const { children } = this.props;
     const { allTheWayLeft, allTheWayRight } = this.state;
     const navClasses = cx({ [`${CN}__nav`]: true });
-    const leftNavClasses = cx({
-      [`${CN}__left-nav`]: true,
-      [`${CN}__nav-disabled`]: allTheWayLeft
-    }, navClasses);
-    const rightNavClasses = cx({
-      [`${CN}__right-nav`]: true,
-      [`${CN}__nav-disabled`]: allTheWayRight
-    }, navClasses);
+    const leftNavClasses = cx(
+      {
+        [`${CN}__left-nav`]: true,
```

Comment:
if there is a class, that should always be added- pass it as argument in cx before object
cx(`${CN}__left-nav`, ....

Reply:
done)